modules/stars.py

```diff
@@ -20,8 +20,263 @@
+                    $1,
+                    $2,
+                    $3,
+                    $4,
+                    $5
+                )"""
+        await self.pool.execute(query, message_id, bot_message_id, payload_channel_id, reactions, content_id)
+
+    async def add_starer(self, user_id: int, message_id: int):
+        query = """
+                INSERT INTO starers VALUES (
+                    $1,
+                    $2
+                )"""
+
+        await self.pool.execute(query, user_id, message_id)
+
+    async def remove_starer(self, message_id: int, user_id: int):
+        query = """DELETE FROM starers WHERE msg_id = $1 AND user_id= $2"""
+        await self.pool.execute(query, message_id, user_id)
+
+    async def update_entry(self, reactions: int, message_id: int):
+        query = """UPDATE starboard_entries SET stars = $1 WHERE msg_id = $2"""
+        await self.pool.execute(query, reactions, message_id)
+
+    async def remove_entry(self, message_id: int):
+        query = """DELETE FROM starboard_entries WHERE msg_id= $1"""
+        await self.pool.execute(query, message_id)
+
+    async def clear_starers(self, message_id: int):
+        query = """DELETE FROM starers WHERE msg_id = $1"""
+        await self.pool.execute(query, message_id)
+
+    def get_formatted_time(self):
+        now = datetime.datetime.now()
+        time = now.strftime("%m/%d/%Y %I:%M %p")
+        return time
+
+    async def handle_star(self, payload: discord.RawReactionActionEvent):
+        time = self.get_formatted_time()
+        entry: StarboardEntry = StarboardEntry(self.pool, payload.message_id)
+        await entry.fetch()
+
+        if str(payload.emoji) != STARBOARD_EMOJI:
+            return
+
+        channel: discord.TextChannel = self.bot.get_channel(payload.channel_id)  # type: ignore
+        message: discord.Message = await channel.fetch_message(payload.message_id)
+
+        reaction = discord.utils.get(message.reactions, emoji=STARBOARD_EMOJI)
+        reaction_count = reaction.count if reaction else 0
+
+        if entry.exists:
+            bot_msg_id = entry.bot_message_id
+
+            query = """SELECT * FROM starers WHERE user_id={} AND msg_id={}""".format(payload.user_id, entry.msg_id)
+
+            starer = await self.pool.fetchrow(query)
+
+            if starer is not None:
+                return
+
+            await self.add_starer(payload.user_id, payload.message_id)
+
+            bot_channel: discord.TextChannel = self.bot.get_channel(self.starboard_channel_id)  # type: ignore
+            bot_message = await bot_channel.fetch_message(bot_msg_id)
+
+            stars = reaction_count
+            star = self.get_star(stars)
+            await bot_message.edit(content=HEADER_TEMPLATE.format(star, stars, payload.channel_id, payload.channel_id))
+            await self.update_entry(stars, payload.message_id)
+            return
+
+        if not reaction_count >= self.entry_requirement:
```

Comment:
```suggestion
        if reaction_count < self.entry_requirement:
```
Could be simplified?